src/lm_saes/config.py

```diff
@@ -254,6 +261,8 @@ class ActivationFactoryConfig(BaseConfig):
         else None
     )
     """ Buffer size for online shuffling. If `None`, no shuffling will be performed. """
+    buffer_shuffle_config: Optional[BufferShuffleConfig] = None
```

Comment:
Seems better to rename the field name `buffer_shuffle_config` to `buffer_shuffle` since it's already wrapped in a config.